bookshelf/app.yaml.dist

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
Well, wait. On the second thought, it is much easier for us to keep it `app.yaml.dist`, especially during our development. I will keep it for now.